src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -433,6 +439,29 @@ def run_tests_in_chroot(self):
     """
     return self.get_options().chroot
 
+  @staticmethod
+  def _copy_files(dest_dir, target):
+    if isinstance(target, Files):
+      for source in target.sources_relative_to_buildroot():
+        src = os.path.join(get_buildroot(), source)
+        dest = os.path.join(dest_dir, source)
+        safe_mkdir_for(dest)
+        shutil.copy(src, dest)
+
+  @contextmanager
+  def _chroot(self, targets, workdir):
```

Comment:
Since you're now calling this in a subclass, it should lose the underscore prefix. We reserve Python's faux-private convention for symbols used only in the same class/file. The idea being that you shouldn't have to look outside the files for uses when making changes.

Reply:
Ok -- that sounds like we should also lose the underscore on `_spawn_and_wait()` then?

Reply:
Did both in 5d9b63c!

Reply:
Thanks! I wouldn't have asked you to fix `_spawn_and_wait`, as that's a preexisting condition... But thanks for doing it anyway.

Reply:
Oh -- is it better to isolate this change to `_chroot` then (just as a matter of what to do in these cases)? I'm fine with that!